lib/provisioner/worker/plugins/automators/chef_solo_automator/resources/cookbooks/coopr_base/attributes/default.rb

```diff
@@ -17,6 +17,11 @@
 # limitations under the License.
 #
 
+if node.key?('base')
+  Chef::Log.warn('Old "base" attributes found! Converting to "coopr_base"... use node[:coopr_base] going forward!')
+  node.default['coopr_base'] = node['base'].merge(node['coopr_base'])
```

Comment:
you sure this works?  pretty sure this will fail if `node['coopr_base']` doesn't exist yet.  (``{}.merge(nil)``)